components/clarity-vscode/README.md

```diff
@@ -1,50 +1,69 @@
 
-You'll need to have Rust, Node.js and NPM installed.
+When you select a function, the extension adds the necessary parentheses around it and puts placeholders in the arguments of the function.
 
-From the `./components/clarity-vscode`, run `npm install` to install the dependencies and `npm run dev` to start the extension in a Chromium instance.
+![autocomplete gif](images/autocomplete.gif)
 
-### Structure
+### Check Contract on Save and Display Errors Inline
 
-The LSP has two main parts: the client and the server.
-This two part will run differents environments:
-- VSCode Web: in a WebWorkers
-- VSCode Desktop: in a Node.js environments
+When a contract is opened or saved, the extension will notify you if errors are found (syntax, unknown keyword, etc), or warnings (such as unsafe code). This helps you to ensure that you write safe and clean code.
 
-The LSP (`./components/clarity-lsp`), written in Rust, is built with wasm-pack for both these environments and will be load accordingly, with `fetch()` in the browser and `fs.readFileSync()` in Node.js.
+![display errors gif](images/errors.gif)
 
-```
-./components/clarity-vscode
-├── package.json // The extension manifest.
-├── client
-│   └── src
-│       ├── clientBrowser.ts
-│       ├── clientNode.ts
-│       └── common.ts
-└── server
-    └── src
-        ├── serverBrowser.ts
-        ├── serverNode.ts
-        └── common.ts
-```
+### VS-Code Support
+
+Visual Studio Code is natively supported, but this extension provides additional support for [vscode.dev](https://vscode.dev/), [github.dev](https://github.dev/github/dev) and similar tools.
+
+For more information, please see the [Clarinet GitHub VS Code documentation page](https://github.com/hirosystems/clarinet/blob/develop/components/clarity-vscode/README.md)
+
+### Auto Complete User-Defined Functions
+
+This feature is similar to the auto-complete feature, but does not provide documentation for functions when writing a contract.
+### Resolve Contract-Call Targeting Local Contracts
+
+This feature is similar to the auto-complete feature, but enables auto-completion over multiple files.
+
+### Support for Multiple Errors
+
+Although not considered a native feature for the Visual Studio Code extension, support for multiple errors is now supported in this version.
+
+![multiple error support](images/multicontract.gif)
+### Support for Traits
+
+When a contract implements a trait (such as the NFT of FT trait – SIPs 009 and 010), the extensions will show and errors if the trait implementation is not satisfied (for example, if the trait expects a function which is not implemented, or if the function signature does not match the trait definition).
+### Documentation
+
+This feature extends the capabilities of the native auto-complete function by providing documentation for a function when writing a smart contract.
+
+### Debugger
```

Comment:
I would put it higher in the list importance.
Also, we need to mention that in only works on Desktop (at least for now) in that it requires a local installation of clarinet (cf [here](https://github.com/hirosystems/clarinet/pull/571/files#diff-48b9d6c32006f08d5050dbfb8dd4fe9d13005db94738ce97509b0e96c203f7f6L17))

Reply:
Lets, please put the local installation of Clarinet wrapped inside a NOTE or with BOLD formatting so it is front and center and folks know that it is a prerequisite.

Reply:
I added note in bold stating that this requires a local installation of Clarinet. I also moved this section up in the list of features so it is called out earlier.